charts/datadog/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Datadog changelog
 
+## 2.6.5
+
+* Remove the cluster-id configmap mount for process-agent. This requires an Agent version >= 7.25 and a Cluster Agent >= 1.11
```

Comment:
I’m wondering if bumping the minimal version of the agent and the cluster agent deserves a bump of the minor version instead of the patch version of the Helm chart.
Especially since the default in `values.yaml` are not meeting those requirements yet.
https://github.com/DataDog/helm-charts/blob/master/charts/datadog/values.yaml#L561
https://github.com/DataDog/helm-charts/blob/master/charts/datadog/values.yaml#L360